src/index.html

```diff
@@ -10,8 +10,137 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="field-group">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:
+          <input class="form-controls"
+          type="text"
+          autocomplete="off"
+          name="surname"
+          minlength="2"
+          maxlength="30"
+          required
+          >
```

Comment:
the first attribute also has to be on the new line. Attributes have to be indented.

```suggestion
          <input
            class="form-controls"
            type="text"
            autocomplete="off"
            name="surname"
            minlength="2"
            maxlength="30"
            required
          >
```